src/main/java/core/basesyntax/service/ShopServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.FruitOperationDao;
+import core.basesyntax.model.FruitOperation;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.List;
+
+public class ShopServiceImpl implements ShopService {
```

Comment:
The class `ShopServiceImpl` should not have an empty line at the beginning of its implementation according to the checklist. Please remove the redundant empty line.